compoments/report-repair/repair-image-upload.js

```diff
@@ -5,6 +5,34 @@ import Button from '../button';
 import {serviceName} from '../../helpers/constants';
 import imageToBase64 from 'image-to-base64/browser';
 import {isMvpReleaseVersion} from '../../helpers/features';
+import { imageValidator } from '../../helpers/validators';
+
+export const ImagePreview =({image, onDelete}) => {
```

Comment:
I like the way you've exported ImagePreview here so we don't duplicate it in the other page.

Reply:
Thanks! I've made one more change & moved it to a separate component as suggested by Bhavesh.